platform.md

```diff
@@ -396,57 +437,14 @@ When analyzing a given layer the lifecycle SHALL:
     - Write layer metadata read from the cache to `<layers>/<buildpack-id>/<layer-name>.toml`
     - Write the layer diffID from the cache to `<layers>/<buildpack-id>/<layer-name>.sha`
 
-#### `restorer`
-Usage:
-```
-/cnb/lifecycle/restorer \
-  [-cache-dir <cache-dir>]
-  [-cache-image <cache-image>]
-  [-gid <gid>] \
-  [-group <group>] \
-  [-layers <layers>] \
-  [-log-level <log-level>] \
-  [-uid <uid>]
-```
-
-##### Inputs
-| Input          | Environment Variable  | Default Value         | Description
-|----------------|-----------------------|-----------------------|----------------------
-| `<cache-dir>`  | `CNB_CACHE_DIR`       |                       | Path to a cache directory
-| `<cache-image>`| `CNB_CACHE_IMAGE`     |                       | Reference to a cache image in an OCI image registry
-| `<gid>`        | `CNB_GROUP_ID`        |                       | Primary GID of the stack `User`
-| `<group>`      | `CNB_GROUP_PATH`      | `<layers>/group.toml` | Path to group definition (see [`group.toml`](#grouptoml-toml))
-| `<layers>`     | `CNB_LAYERS_DIR`      | `/layers`             | Path to layers directory
-| `<log-level>`  | `CNB_LOG_LEVEL`       | `info`                | Log Level
-| `<uid>`        | `CNB_USER_ID`         |                       | UID of the stack `User`
-| `<layers>/<buidpack-id>/<layer>.sha`  ||                       | Files containing the diffID of each analyzed layer
-| `<layers>/<buidpack-id>/<layer>.toml` ||                       | Files containing the layer content metadata of each analyzed layer (see data format in [Buildpack Interface Specification](buildpack.md))
-
-##### Outputs
-| Output                             | Description
-|------------------------------------|----------------------------------------------
-| [exit status]                      | (see Exit Code table below for values)
-| `/dev/stdout`                      | Logs (info)
-| `/dev/stderr`                      | Logs (warnings, errors)
-| `<layers>/<buidpack-id>/<layer>/*` | Restored layer contents
-
-| Exit Code | Result|
-|-----------|-------|
-| `0`       | Success
-| `11`      | Platform API incompatibility error
-| `12`      | Buildpack API incompatibility error
-| `1-10`, `13-99` | Generic lifecycle errors
-| `300-399` | Restoration-specific lifecycle errors
-
-##### Layer restoration
 For each layer metadata file found in the `<layers>` directory, the lifecycle:
 - MUST restore cached layer contents if the cache contains a layer with matching diffID
 - MUST remove layer metadata if `cache=true` AND the cache DOES NOT contain a layer with matching diffID
```

Comment:
Instead of require the `restorer` to write the layer metadata and then remove it. We should combine these lines with the lines above and say that the restorer must not write the layer memtadata in these cases.